modules/installation-vsphere-regions-zones.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+//* installing/installing-vsphere-installer-provisioned-customizations.adoc [IPI]
+//* installing/installing-vsphere-installer-provisioned-network-customizations.adoc [IPI]
+//* installing/installing-vsphere.adoc [UPI]
+//* installing/installing-vsphere-network-customizations.adoc [UPI]
+//* installing/installing-restricted-networks-installer-provisioned-vsphere.adoc [IPI]
+//* installing/installing-restricted-networks-vsphere.adoc [IPI]
+
+:_content-type: CONCEPT
+[id="installation-vsphere-regions-zones_{context}"]
+= VMware vSphere region and zone enablement
+
+You can deploy an {product-title} cluster to multiple vSphere datacenters that run in a single VMware vCenter. Each datacenter can run multiple clusters. This configuration reduces the risk of a hardware failure or network outage that can cause your cluster to fail.
+
+:FeatureName: VMware vSphere region and zone enablement
+include::snippets/technology-preview.adoc[]
+
+The default installation configuration deploys a cluster to a single vSphere datacenter. If you want to deploy a cluster to multiple vSphere datacenters, you must create an installation configuration file that enables the region and zone feature.
+
+The default `install-config.yaml` file includes `vcenters` and `failureDomains` fields, where you can specify multiple vSphere datacenters and clusters for your {product-title} cluster. You can leave these fields blank if you want to install an {product-title} cluster in a vSphere environment that consists of single datacenter.
+
+The following list describes terms associated with defining zones and regions for your cluster:
+
+* Failure domain: Establishes the relationships between a region and zone. You define a failure domain by using vCenter objects, such as a `datastore` object. A failure domain defines the vCenter location for {product-title} cluster nodes.
+* Region: Specifies a vCenter datacenter. You define a region by using a tag from the  `openshift-region` tag category.
+* Zone: Specifies a vCenter cluster. You define a zone by using a tag from the `openshift-zone` tag category.
+
+[NOTE]
+====
+If you plan on specifying more than one failure domain in your `install-config.yaml` file, you must create tag categories, zone tags, and region tags in advance of creating the configuration file.
+====
+
+You must create a vCenter tag for each vCenter datacenter, which represents a region. Additionally, you must create a vCenter tag for each cluster than runs in a datacenter, which represents a zone. After you create the tags, you must attach each tag to their respective datacenters and clusters.
+
+The following table outlines an example of the relationship among regions, zones, and tags for a configuration with multiple vSphere datacenters running in a single VMware vCenter.
+
+.Example of a configuration with multiple vSphere datacentres that run in a single VMware vCenter
```

Comment:
```suggestion
.Example of a configuration with multiple vSphere datacenters that run in a single VMware vCenter
```

Reply:
^ To be consistent with paragraph above

Reply:
Thanks, @kcarmichael08 . I let a UK-spelling variant fall in there. I fixed this typo.